spec/models/post_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.create(name: 'John Doe')
+  end
+
+  let(:myPost) do
+    Post.new(
+      title: 'Hello',
+      text: 'This is my first post',
+      comments_counter: 3,
+      likes_counter: 1,
+      author: user
+    )
+  end
+
+  describe 'Validations' do
+    it 'Should return not valid for a blank title' do
+      myPost.title = nil
+      expect(myPost).to_not be_valid
+    end
+
+    it 'Should return not valid for a title more than 250 chars' do
+      myPost.title = 'Lorem blalllllllllllllllllllllllllllll ipsum dolor sit amet, ' \
+                     'consectetur adipiscing elit. Sed auctor vestibulum velit, ' \
+                     'ut eleifend ante mattis in. Maecenas hendrerit, urna nec ' \
+                     'laoreet bibendum, velit dolor ultricies lorem, quis fermentum ' \
+                     'odio massa nec dui.'
```

Comment:
- [Optional :+1: ] Situations like this may arises where you are requested to replicate a string of characters several time. The good news is that you can do it with `*` operator :+1: (use between a string and an integer) commonly refer to as _string replicator operator_--repeat a string many times. Ex: `"Lorem" * 240`. I think you'll love to test this out. This may render your codes clean